diff -urN gtetrinet-0.7.7.orig/configure.in gtetrinet-0.7.7/configure.in
--- gtetrinet-0.7.7.orig/configure.in	2004-04-18 21:33:32.525865312 +0200
+++ gtetrinet-0.7.7/configure.in	2004-04-18 21:33:46.154793400 +0200
@@ -12,7 +12,7 @@
 AC_PROG_CC
 
 AC_PROG_INTLTOOL([0.25])
-ALL_LINGUAS="am az ca cs cy de en_CA en_GB es eu fr ga hr ja mk ml ms nl no pl pt pt_BR ru sr sr@Latn sv uk zh_TW"
+ALL_LINGUAS="am az ca cs cy de en_CA en_GB es eu fr ga hr ja mk ml ms nb nl pl pt pt_BR ru sr sr@Latn sv uk zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 AC_PATH_PROG(GCONFTOOL, gconftool-2)
